test/linter/test-experimental.js

```diff
@@ -0,0 +1,62 @@
+const chalk = require('chalk');
+
+/**
+ * @typedef {import('../../types').Identifier} Identifier
+ * @typedef {import('../utils').Logger} Logger
+ */
+
+/**
+ * @param {Identifier} data
+ * @param {Logger} logger
+ */
+function checkStatus(data, logger, path = []) {
+  const status = data.__compat?.status;
```

Comment:
This is invalid syntax.  I think you were trying to do something like the following?

```suggestion
  if (!data.__compat) return;
  const status = data.__compat.status;
```

Reply:
Oops, please Node 14+ everywhere 😂😂

Reply:
I just looked into this syntax more -- wow, that is so useful, I never knew this was a thing!